### PR TITLE
Deal with ASyTop error

### LINKED ISSUE #14: Add a handler to the base work chain that deals with ASyTop error.

### PR DESCRIPTION
**Pull request by yakutovicha**
fixes #14 

Add an optional handler that deals with the ASyTop error.
Fix the error by rounding the atomic coordinates to the 4th digit.

### CODE REVIEW

File: aiida_gaussian/workchains/base.py [hunk 1]
```diff
@@ -99,6 +100,19 @@ def handle_scf_failure(self, node):
 
         return ProcessHandlerReport(True)
 
+    @process_handler(
+        priority=500, exit_codes=[GaussianCalculation.exit_codes.ERROR_ASYTOP], enabled=False
+    )
+    def handle_asytop_error(self, node):
+        """Handle the error code 500 (ASYTOP)."""
```
Comment (yakutovicha) [inline aiida_gaussian/workchains/base.py]:
```suggestion
        """Handle the error code 302 (ASYTOP)."""
```

File: examples/example_03_base_workflow.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+from aiida.orm import Code, Dict, StructureData
+from aiida.common import NotExistent
+from aiida.plugins import WorkflowFactory
+
+import pymatgen as mg
+
+GaussianBaseWorkChain = WorkflowFactory('gaussian.base')
+
+
+def example_dft(gaussian_code):
```
Comment (eimrek) [inline examples/example_03_base_workflow.py:21]: This example produces the Asytop error, which is correctly handled, right? maybe add a small comment.

Comment (eimrek) [reply examples/example_03_base_workflow.py:21]: ah sorry, it's already there!

**Review by eimrek — APPROVED** [review]
👍

File: examples/example_03_base_workflow.py (hunk 2 above)
Comment (eimrek) [reply examples/example_03_base_workflow.py:21]: But the comment seems to be unfinished

**Review by eimrek — CHANGES_REQUESTED** [review]

File: examples/example_03_base_workflow.py (hunk 2 above)
Comment (yakutovicha) [reply examples/example_03_base_workflow.py:21]: Yes, sorry, not sure how did this happen. Fixed.

**Pull request merged**